src/components/home/sprint-view-card.tsx

```diff
@@ -8,6 +8,11 @@ import UserRoleUtils from "src/utils/user-role-utils";
  */
 const SprintViewCard = () => {
   const adminMode = UserRoleUtils.adminMode();
+  const userIsDeveloper = UserRoleUtils.isDeveloper();
```

Comment:
Is this actually needed? Can't you just use the logic as its set up in the admin-screen and the home-screen and that's enough?